pouchdb-core/pouchdb-core.d.ts

```diff
@@ -268,6 +268,12 @@ declare namespace PouchDB {
         allDocs(options?: Core.AllDocsOptions):
             Promise<Core.AllDocsResponse<Content>>;
 
+        bulkDocs(docs: Core.Document<Content>[],
+                 options: Core.PutOptions | void,
+                 callback: Core.Callback<Core.Error, Core.Response[]>): void;
+        bulkDocs(docs: Core.Document<Content>[],
+                 options?: Core.PutOptions): Promise<Core.Response[]>;
```

Comment:
Thanks. Actually, I can't find where it's said that it's an array.
I was on pouchdb slack and I had the confirmation from someone that it's just an object.
In addition, this is some go implementation:
[Go implementation of pouchdb](https://godoc.org/github.com/flimzy/go-pouchdb#PouchDB.BulkDocs)